github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -533,11 +533,13 @@ presubmits:
         env:
         - name: TARGET
           value: windows2016
+        - name: KUBEVIRT_MEMORY_SIZE
+          value: 8G
         image: quay.io/kubevirtci/bootstrap:v20220512-78048f1
         name: ""
         resources:
           requests:
-            memory: 29Gi
+            memory: 32Gi
```

Comment:
An increase of memory here may lead to it being more difficult to schedule windows jobs as the memory request will be higher than the other e2e jobs.